### ISSUE TITLE
cannot decrypt secrets

### ISSUE THREAD

**Issue by divisionx3** [jkroepke/helm-secrets#125]
Hello all,

for a few weeks I have been evaluating ways to encrypt secrets for Kubernetes / Helm in order to be able to push them into our git repository without entering into security issues. I noticed this fork from helm secrets positively. However, I currently have problems decrypting the secret files. To describe it exactly, an example:

helm secrets enc secrets.yml <-- Secret is properly encrypted
helm secrets view secrets.yml <-- Displays the decrypted secrets
helm secrets dec secrets.yml <-- Throws an error:

helm secrets dec secrets.yml
[helm-secrets] Decrypting secrets.yml
Incorrect Usage. flag provided but not defined: -output

The same applies to the following:

helmet secrets upgrade name . -f secrets.yml

[helm-secrets] Decrypt: secrets.yml
Error: open secrets.yml: no such file or directory

rm: cannot remove 'secrets.yml.dec': No such file or directory
Error: plugin "secrets" exited with error

Unfortunately, I couldn't find anything to solve this problem anywhere. Also in the sops documentation I did not find any clues.

Since I would love to use the plugin, I hope that anyone can help me.

**Comment by jkroepke** [comment]
Hi @divisionx3,

did you wrote a mail to me days ago? I got a mail have the same issue as described in my inbox, but this is the correct place for issues.

> Incorrect Usage. flag provided but not defined: -output

Thats strange, I dont know, why `-output` is used here.

I can verify, this message comes from:

```
jkr@joe-nb infra-az-dev-svc % sops -v    
sops 3.7.1 (latest)
jkr@joe-nb infra-az-dev-svc % sops -output
Incorrect Usage. flag needs an argument: -output
```
But helm-secrets clearly use `--output` here

https://github.com/jkroepke/helm-secrets/blob/5a95b0b27dda5e5e1ec97de5baede875dd06b2c0/scripts/drivers/sops.sh#L25

Whats you version of sops?
Whats you version of helm-secrets?
Whats your operation system?
Can you verify, your local source code is not modified?

**Comment by divisionx3** [comment]
Hello @jkroepke,

yes i wrote the mail to you. 

My settings are the following:

**OS:** 
debian

**SOPS: ii  sops                                 3.0.4                           amd64        Sops is an editor of encrypted files that supports YAML, JSON** and BINARY formats and encrypts with AWS KMS and PGP.

**helm**
version.BuildInfo{Version:"v3.5.3", GitCommit:"041ce5a2c17a58be0fcd5f5e16fb3e7e95fea622", GitTreeState:"dirty", GoVersion:"go1.15.8"}

**helm secrets:** 
secrets 3.8.1   This plugin provides secrets values encryption for Helm charts secure storing

And yes my code is not modified.

**Comment by divisionx3** [comment]
Hi,

i fixed the issue by using a newer sops version.

Thank you very much!

**Comment by jkroepke** [comment]
Correct, the output flags is available in sops 3.2.0+

https://github.com/mozilla/sops/commit/062714df8fba88a9d7a3c418388b07bdb92cc8cf

But you already find out, that a update helps here.
